src/lib/split-view/split-view/_mixins.scss

```diff
@@ -0,0 +1,42 @@
+@use './variables';
```

Comment:
The `_variables.scss` file is empty, do you have anything to move in there? If not, let's just remove the file for now.